pkgs/development/r-modules/default.nix

```diff
@@ -939,6 +938,39 @@ let
   ];
 
   otherOverrides = old: new: {
+    # arrow 14.0.0.2 on CRAN is lagging behind current libarrow release number
+    # https://github.com/apache/arrow/issues/39698
+    arrow = let
+      arrow-cpp = pkgs.arrow-cpp.overrideAttrs (finalAttrs: previousAttrs: {
+        version = "14.0.0";
+        src = fetchurl {
+          url = "mirror://apache/arrow/arrow-${finalAttrs.version}/apache-arrow-${finalAttrs.version}.tar.gz";
+          hash = "sha256-TrDaUOwHG68V/BY8tIBYkx4AbxyGLI3vDhgP0H1TECE=";
+        };
+    });
+    in old.arrow.overrideAttrs (attrs: {
+      postPatch = ''
+        patchShebangs configure
+      '';
+      # OpenSSL and CURL to support reading AWS S3 and Google Cloud Storage
+      # R is needed at compile time; otherwise: "> /nix/store/zf8hff9bnlfix7l7gjsnv3jc22lxzzxx-stdenv-darwin/setup: line 1593: R: command not found"
+      nativeBuildInputs = [
+        pkgs.pkg-config
+        pkgs.cmake
+        pkgs.curl
+        pkgs.openssl
+        arrow-cpp
+        pkgs.R
+      ] ++  lib.optionals stdenv.isDarwin [ pkgs.intltool ];
+      buildInputs = [
```

Comment:
Are these needed in buildinputs? If yes, are they needed in nativebuildinputs? I doubt that they need to be in both.